books/RayTracingTheNextWeek.html

```diff
@@ -187,27 +145,33 @@
     class moving_sphere : public hittable {
       public:
         moving_sphere() {}
-        moving_sphere(
-            point3 ctr0, point3 ctr1, double r, shared_ptr<material> m,
-            double time_start, double time_end)
-            :
-            center0(ctr0), center1(ctr1), radius(r), mat_ptr(m),
-            time0(time_start), time1(time_end)
-        {};
+        moving_sphere(point3 c0, point3 c1, double r, shared_ptr<material> m)
+          : center0(c0), center1(c1), center_vec(c1 - c0), radius(r), mat_ptr(m)
+        {
+            // To Be Implemented
```

Comment:
Remove. Change to:

```
        moving_sphere(point3 c0, point3 c1, double r, shared_ptr<material> m)
          : center0(c0), center1(c1), center_vec(c1 - c0), radius(r), mat_ptr(m)
          {}
```

The constructor is complete as far as this chapter is concerned.